dask_ml/wrappers.py

```diff
@@ -662,9 +663,28 @@ def _get_output_dask_ar_meta_for_estimator(model_fn, estimator, input_dask_ar):
     """
     # sklearn fails if input array has size size
     # It requires at least 1 sample to run successfully
-    ar = np.zeros(
-        shape=(1, input_dask_ar.shape[1]),
-        dtype=input_dask_ar.dtype,
-        like=input_dask_ar._meta,
-    )
+    imput_meta = input_dask_ar._meta
+    if hasattr(imput_meta, "__array_function__"):
+        ar = np.zeros(
+            shape=(1, input_dask_ar.shape[1]),
+            dtype=input_dask_ar.dtype,
+            like=imput_meta,
+        )
+    elif "scipy.sparse" in type(imput_meta).__module__:
+        # sparse matrices dont support
+        # `like` due to non implimented __array_function__
+        # Refer https:/q/github.com/scipy/scipy/issues/10362
+        # Note below works for both cupy and scipy sparse matrices
+        ar = type(imput_meta)((1, input_dask_ar.shape[1]), dtype=input_dask_ar.dtype)
+    else:
```

Comment:
When does this clause get triggered? Could we have a test for it?

Reply:
Ah, I see from your todo list you're planning on adding a test, that's great!

Reply:
Yup. Added a test [here](https://github.com/dask/dask-ml/blob/48645448147126c6a343d6d085dd571be0cd1665/tests/test_parallel_post_fit.py#L253-L291) . 

This basically gets triggered when using dask array wrapper around an object that does not support `__array_function__` .   A real world common example will be a Tensorflow tensor